packages/collector/src/announceCycle/unannounced.js

```diff
@@ -220,3 +222,29 @@ function applySpanBatchingConfiguration(agentResponse) {
     agentOpts.config.tracing.spanBatchingEnabled = true;
   }
 }
+/**
+ * @param {AgentAnnounceResponse} agentResponse
+ */
+function applyIgnoreEndpointsConfiguration(agentResponse) {
+  if (agentResponse?.tracing?.['ignore-endpoints']) {
+    const endpointTracingConfigFromAgent = agentResponse.tracing['ignore-endpoints'];
+
+    const endpointTracingConfig = Object.fromEntries(
+      Object.entries(endpointTracingConfigFromAgent).map(([service, endpoints]) => {
+        let normalizedEndpoints = null;
+        if (typeof endpoints === 'string') {
```

Comment:
The agent configuration currently uses pipe ('|') as a separator for endpoints, We split the string by both pipe and comma to support these format internally. Additionally, we support string[] for backward compatibility, as string[] was the standard followed previously.

Reply:
Thats a good idea. Could you please make that clear in a comment on the fn header?